.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: monthly
+    time: "03:00"
+  commit-message:
+    prefix: '[bot]'
```

Comment:
Previously there were no prefix in Dependabot's commit messages, but I will suggest to have `[bot]` as a prefix.